types/fetch.d.ts

```diff
@@ -42,7 +42,7 @@ export interface BodyMixin {
   readonly text: () => Promise<string>
 }
 
-export type HeadersInit = string[][] | Record<string, string> | Headers
+export type HeadersInit = string[][] | Record<string, string | string[]> | Headers
```

Comment:
Should it be `Readonly<string>`? I think the array is not mutated?

Reply:
It shouldn't really make any much difference, and I also just kept the same behavior it was using on the same line (no `Readonly` on `string[][]`)

Reply:
The difference is that it allows to pass a readonly array. It was a non-blocking suggestion, though.

Reply:
Makes sense, less restrictive. Should I also adjust `string[][]`?